### PR TITLE
Updated german translation for v. 1.11.0

### PR DESCRIPTION
**Pull request by tweimer**
- Includes changes for #933
- Fixes some minor typos

See #306

### CODE REVIEW

File: resources/language/resource.language.de_de/strings.po [hunk 1]
```diff
@@ -325,7 +325,7 @@ msgstr "Audio- und Untertitel-Einstellungen beibehalten"
 
 msgctxt "#30083"
 msgid "Force the display of forced subtitles only with the audio language set"
-msgstr ""
+msgstr "Anzeige erzwungener Untertitel nur erzwingen wenn Audiosprache gesetzt ist"
```
Comment (Maven85) [inline resources/language/resource.language.de_de/strings.po:328]: There is a comma missing after "erzwingen".

File: resources/language/resource.language.de_de/strings.po [hunk 2]
```diff
@@ -692,7 +692,7 @@ msgstr "Aktiviere Workaround für STRM-Wiedergabe-Fortsetzung"
 
 msgctxt "#30177"
 msgid "Ask to resume the video"
-msgstr "Frage zum Fortsetzen des Videos"
+msgstr "Fragen ob Video fortgesetzt werden soll"
```
Comment (Maven85) [inline resources/language/resource.language.de_de/strings.po:695]: There is a comma missing after "Fragen".

File: resources/language/resource.language.de_de/strings.po [hunk 3]
```diff
@@ -708,7 +708,7 @@ msgstr "Es gab ein Problem bei der Anmeldung, möglicherweise ist Ihr Account ni
 
 msgctxt "#30181"
 msgid "Always show the subtitles when the preferred audio language is not available"
-msgstr ""
+msgstr "Untertitel immer anzeigen wenn bevorzugte Audiosprache nicht verfügbar ist"
```
Comment (Maven85) [inline resources/language/resource.language.de_de/strings.po:711]: There is a comma missing after "anzeigen".

File: resources/language/resource.language.de_de/strings.po (hunk 1 above)
Comment (tweimer) [reply resources/language/resource.language.de_de/strings.po]: Thanks. fixed!

File: resources/language/resource.language.de_de/strings.po (hunk 3 above)
Comment (tweimer) [reply resources/language/resource.language.de_de/strings.po]: Thanks. fixed!

File: resources/language/resource.language.de_de/strings.po (hunk 2 above)
Comment (tweimer) [reply resources/language/resource.language.de_de/strings.po]: Thanks. fixed!

**Pull request merged**